openedx/core/djangoapps/notifications/tests/test_views.py

```diff
@@ -118,18 +116,33 @@ def setUp(self):
             is_active=True,
             mode='audit'
         )
-        self.post_save_signal = Signal()
+        # self.post_save_signal = Signal()
```

Comment:
can we remove this, if this is not needed?

Reply:
Done